frontend/src/InputForm.js

```diff
@@ -634,6 +634,7 @@ class InputForm extends React.Component {
     }).then((response) => response.json())
       .then((responseSchedules) => {
         this.props.setScheduleList(responseSchedules);
+        this.props.setTermDates(this.state.basicInfo.termDates.startDate, this.state.basicInfo.termDates.endDate);
```

Comment:
nit: improve legibility here by destructuring the state first:

```
const { startDate, endDate } = this.state.basicInfo.termDates;

...

this.props.setTerDates(startDate, endDate);
```

or something similar